packages/core/src/__tests__/getStateFromPath.test.tsx

```diff
@@ -2196,6 +2196,7 @@ it('tries to match wildcard patterns at the end', () => {
             path: '/bar/:id/',
             screens: {
               404: '*',
+              matches: ':params',
```

Comment:
Looks like a leftover?

Reply:
No, it's meant to be here. 

that `:params` should take lower priority compare to 'test'. Just like '*'?

Reply:
Since `:params` and `*` are both wildcards

Reply:
Can you rename `matches` to some name like `Test1` etc.? `matches` looks like it's some option and not a screen name